src/routes/pageHeader.svelte

```diff
@@ -0,0 +1,94 @@
+<script>
+    import { goto } from "$app/navigation";
+    import { onMount } from "svelte";
+    import { TabGroup, Tab, TabAnchor } from "@skeletonlabs/skeleton"; // Menu with tabs
+    import { AppBar } from "@skeletonlabs/skeleton";
+    import { page } from "$app/stores"; // Contains all pages in a store
+
+    // Transient fault handling library import
+    import { ConstantBackoff, handleAll, retry } from "cockatiel";
+    import { urls } from "$lib/util.js";
+
+    // Definitions
+    let loggedIn = false;
+
+    // Define retry policy
+    const retryPolicy = retry(handleAll, {
+        maxAttempts: 3, // Try 3 times
+        backoff: new ConstantBackoff(50), // Wait 10ms after each try
+    });
+
+    // Functions
+    async function redirectToLogin() {
+        goto("/login");
+    }
+
+    // Runs as soon as the component is mounted
+    onMount(async () => {
+        try {
+            console.log("logged");
+            // Fetch backend to check if user is signed in
+            const response = await retryPolicy.execute(() =>
+                fetch(urls.get.isLoggedIn, { credentials: "include" }),
+            );
+            if (response.ok) {
+                loggedIn = true;
+            } else {
+                loggedIn = false;
+            }
+        } catch (err) {
+            console.log(err);
+        }
+    });
+</script>
+
+<div>
+    <AppBar
+        gridColumns="grid-cols-3"
+        slotDefault="place-self-center"
+        slotTrail="place-content-end"
+    >
+        <svelte:fragment slot="lead">
+            <img src="/menuIcon.svg" alt="menu icon" />
+        </svelte:fragment>
+        <svelte:fragment slot="default">
+            <a href="/">
+                <!-- Voycar Logo -->
+                <img
+                    src="/logo-full-white.svg"
+                    alt="logo"
+                    class="w-500 scale-50"
+                />
+            </a>
+        </svelte:fragment>
+        <svelte:fragment slot="trail">
+            {#if loggedIn}
+                <a href="/user">
+                    <!-- User icon -->
+                    <img src="/userIcon.svg" alt="user icon" />
+                </a>
+            {:else}
+                <button
+                    class="variant-ringed-primary btn"
+                    on:click={() => redirectToLogin()}>Anmelden</button
```

Comment:
```suggestion
                    on:click={redirectToLogin}>Anmelden</button
```
Ich glaube das geht auch so, bitte einmal testen